errors.yaml

```diff
@@ -66,4 +66,8 @@ errors:
     description: "The requested database has reached its maximum size; no further documents can be added. For information on customizing the maximum database size accepted by MeiliSearch, check out this [guide](https://docs.meilisearch.com/reference/features/configuration.html#max-udb-size)."
   - code: index_primary_key_already_exists
     description: "The requested index already has a primary key that cannot be changed."
+  - code: invalid_token
```

Comment:
This error has been replaced by `invalid_api_key` in the last release. No need to use `invalid_token` now 😇